watcher.go

```diff
@@ -307,7 +303,10 @@ func (w *Watcher) Recaller(n Notifier) Recaller {
 	return func(dep dep.Dependency) (interface{}, bool) {
 		w.register(n, dep)
 		data, ok := w.cache.Recall(dep.String())
-		if !ok {
+		switch {
+		case ok:
+			w.tracker.dataUsed(n, dep)
```

Comment:
The name `dataUsed` in relation to `cache.Recall` here is not very intuitive for me. It took me following code to the tracker -> trackedPair -> comment above dataUsed to realize what it does. Adding comments around the intent here or perhaps renaming `dataUsed` to relate to cache or recalled would be helpful!

Reply:
The above suggestion stems from wanting to intuitive naming since there seems to now be 3 layers of "use",
* Mark deps for a template to clean up unused deps
* TrackedPairs inUse when the dep is active for a template
* TrackedPairs dataUsed when called from cached for the first time

Reply:
Agreed, that wasn't one of my more inspired names. Sometimes I just punt to get on with things hoping something better occurs to me. Nothing ever did.... but I'll see what I can come up with.

Reply:
How about... 'cacheRecalled` or `cacheAccessed`. I kind of like how cacheAccessed reads a bit better, but cacheRecalled ties back to the Recaller which is nice. How do either of those sound?

Reply:
both are improvements 🙌. cacheAccessed works for me

Reply:
Cool. Haven't come up with anything better in the meantime... so `cacheAccessed` it shall be!

Reply:
I'm going to align the other names with it in the same was as inUse does... ie. the trackedPair field will be named the same (cacheAccessed) and the trackPair method will be markCacheAccessed.